mu-plugins/plugin-tweaks/incompatible-plugins.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace WordPressdotorg\MU_Plugins\Plugin_Tweaks\IncompatiblePlugins;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Plugin config.
+ *
+ * Each item in the array contains a plugin to check (`check`), and a plugin
+ * that should be deactivated (`from`) in favor of another plugin (`to`).
+ */
+const PLUGINS = [
+	[
+		// Blocks Everywhere: Uses private/unstable APIs,
+		// which are blocked after GB 16.8.
+		'check' => 'blocks-everywhere/blocks-everywhere.php',
+		'from'  => 'gutenberg/gutenberg.php',
+		'to'    => 'gutenberg-16.8/gutenberg.php',
```

Comment:
We're currently pinned to 16.8.1, is it unnecessary to specify the patch version?

Reply:
👍 Ah of course